src/main/java/com/chopshop166/chopshoplib/outputs/SwPIDSpeedController.java

```diff
@@ -133,90 +100,24 @@ public PIDController getController() {
 
     /** Enable the PID controller. */
     public void enablePID() {
-        if (!enabled) {
-            enabled = true;
+        if (!pidEnabled) {
+            pidEnabled = true;
             dog.enable();
         }
     }
 
     /** Disable the PID controller. */
     public void disablePID() {
-        enabled = false;
+        pidEnabled = false;
         dog.disable();
     }
 
-    @Override
-    public void initSendable(final SendableBuilder builder) {
-        motor.initSendable(builder);
-    }
-
-    @Override
-    public void setP(final double kp) {
-        pid.setP(kp);
-    }
-
-    @Override
-    public void setI(final double ki) {
-        pid.setI(ki);
-    }
-
-    @Override
-    public void setD(final double kd) {
-        pid.setD(kd);
-    }
-
-    @Override
-    public void setF(final double kf) {
-        feedforward = kf;
-    }
-
     @Override
     public void setSetpoint(final double setPoint) {
```

Comment:
I think this should call `SmartMotorController.calculate()` when setting the setpoint.